recipe/conda_build_config.yaml

```diff
@@ -1,6 +1,7 @@
 r_base:
-    - 3.5
-    - 3.6
-    - 4.0
-    - 4.1
-    - 4.2
+  - 3.5
+  - 3.6
+  - 4.0
+  - 4.1
```

Comment:
Is there a compelling reason to support such antiquated R versions? That's not feasible long-term.